request.go

```diff
@@ -380,6 +390,87 @@ func (conn *Connection) ExecuteAsync(expr string, args interface{}) *Future {
 	})
 }
 
+// PrepareAsync sends a sql statement to prepare and returns PreparedStatement object.
+//
+// Since 1.6.0
```

Comment:
Release 1.6.0 was 15 days ago, so you'll need to pick the next tag (I suppose it'll be 1.7.0) [5]

Reply:
Fixed.